supply/lib/supply.rb

```diff
@@ -1,12 +1,12 @@
-require 'json'
```

Comment:
Uh, I don't think we want to change `'` to `"` here and in all the other places.

Reply:
It's `rufo` extension in my vs code. Reverted the changes - https://github.com/fastlane/fastlane/pull/14322/commits/cc4c4efe2d1f40950ab998d00c31206c3b4b136f